source/features/hide-zero-packages.tsx

```diff
@@ -2,16 +2,20 @@ import elementReady from 'element-ready';
 import * as pageDetect from 'github-url-detection';
 
 import features from '.';
+import getTabCount from './remove-projects-tab';
 
-async function init(): Promise<void> {
+async function init(): Promise<void | false> {
 	const packagesCounter = await elementReady([
-		'.numbers-summary [href$="/packages"] .num', // `isRepoRoot`
-		'.UnderlineNav-item[href$="?tab=packages"] .Counter' // `isUserProfile`
-	].join(','));
+		'.numbers-summary [href$="/packages"]', // `isRepoRoot`
+		'.UnderlineNav-item[href$="?tab=packages"]:not(.selected)', // `isUserProfile`
+		'.orgnav > .pagehead-tabs-item[href$="/packages"]:not(.selected)' // `isOrganizationProfile`
+	].join());
 
-	if (packagesCounter?.textContent?.trim() === '0') {
-		packagesCounter.closest('li, .UnderlineNav-item')!.remove();
+	if (!packagesCounter || await getTabCount(packagesCounter) > 0) {
+		return false;
 	}
+
+	(packagesCounter.closest('li, .UnderlineNav-item') ?? packagesCounter).remove();
```

Comment:
Why did you add `?? packagesCounter`? Doesn't seem to ever be needed

Reply:
Organisations the closest will be null

Reply:
Your right, I tested it now, its working just fine